src/telegram_bot/bot_loader.py

```diff
@@ -1,15 +1,30 @@
+from time import sleep
+
+import cv2
 from telegram import Bot, InputMediaPhoto
 from json import dumps
 from src.entities.cat import ClosestCats
-
+from src.telegram_bot.configs.bot_base_configs import S3ClientConfig
+from src.utils.s3_client import YandexS3Client
 
 class DataUploader:
-    def __init__(self, token):
+    def __init__(self, token, answer_time_dely: int, s3_config: S3ClientConfig):
         self.bot = Bot(token)
-
+        self.s3_client = YandexS3Client(s3_config.aws_access_key_id, s3_config.aws_secret_access_key)
+        self.answer_time_dely = answer_time_dely
     def upload(self, closest: ClosestCats):
-        media_group = []
-        for cat in closest.cats:
-            media_group.append(InputMediaPhoto(open(cat.path, "rb"), caption=dumps(cat.additional_info)))
-        self.bot.send_media_group(chat_id=closest.person.chat_id,
+
+        closest_cats_amount = len(closest.cats)
+        self.bot.send_message(closest.person.chat_id, f'We found {closest_cats_amount} similar cats.')
```

Comment:
use not cats but people

Reply:
I can't fix it case my consumer doesnt work without your model. Lets do PR and give me your model